lib/rubocop/cop/style/dir_empty.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Prefer to use `Dir.empty?('path/to/dir')` when checking if a directory is empty.
+      #
+      # @example
+      #   # bad
+      #   Dir.entries('path/to/dir').size == 2
```

Comment:
Btw, probably we should also check for `> 2`, which can be replace with `!Dir.empty?`.